### PR TITLE
MS-960 MS-945 MS-959 MS-955 MS-925 fix: replace forking logic

### PR DESCRIPTION
**Pull request by jjrobothamho**
Replace forking logic with custom behaviour

To ensure back button routing and page navigation is robust.

### CODE REVIEW

File: apps/nrm/index.js [hunk 1]
```diff
@@ -8,6 +8,30 @@ const dataToPdf = require('./behaviours/data-to-pdf');
...
+const reportedToPolice = require('./behaviours/reported-to-police');
+const pvDob = require('./behaviours/pv-dob');
+const pvHoReference = require('./behaviours/pv-ho-reference');
+const pvWantToSubmitNrm = require('./behaviours/pv-want-to-submit-nrm');
+const doesPvNeedSupport = require('./behaviours/does-pv-need-support');
+const pvInterpreterRequirements = require('./behaviours/pv-interpreter-requirements');
+const refuseNrm = require('./behaviours/refuse-nrm');
+const whoContact = require('./behaviours/who-contact');
+const someoneElse = require('./behaviours/someone-else');
+const frAlternateContact = require('./behaviours/fr-alternate-contact');
```
Comment (sulthan-ahmed) [inline apps/nrm/index.js]: Wouldn’t it be more clear if the behaviours were prefixed with backLink

Comment (jjrobothamho) [reply apps/nrm/index.js]: Some behaviours  like pv-gender.js have amendments to the backlink and next step functions and others such as someone-else.js only the backlink was amended.

Should I prefix all the new behaviours with backLink nevertheless? @sulthan-ahmed

Comment (sulthan-ahmed) [reply apps/nrm/index.js]: for the backlink ones prefix with bkLinkWhoContact
for both bkLinkNextSomeoneElse

Comment (jjrobothamho) [reply apps/nrm/index.js]: Thanks

**Comment by sulthan-ahmed** [comment]
restarted the build for you

**Comment by jjrobothamho** [comment]
Removed blank tests and merged.

**Pull request merged**